src/test/java/bio/terra/janitor/service/cleanup/FlightSchedulerTest.java

```diff
@@ -75,7 +95,19 @@ private boolean resourceStateIs(TrackedResourceId id, TrackedResourceState expec
   }
 
   @Test
-  public void resourceScheduledForCleanup() throws Exception {
+  public void resourceScheduledForCleanup_googleBucket() throws Exception {
```

Comment:
I don't think we should be changing the FlightSchedulerTest for buckets.
The scheduler test is about handing things off to Stairway and back. It's unit testing that integration. We don't need to involve real flights for real resources.
We can have a test for the individual cleanup steps, or the whole flight, but we don't need to involve the scheduler to do them.
We can have an end to end test using the Janitor's public APIs to test the whole thing.

Reply:
Those tests are not there, those lives under FlightManager before. They were at FlightManagerTests where I tested that flight. 
Then I found this test failed because it expects state to be ERROR but now they are DONE. So I moved the test here. 
Do you want me to move the cleanup test back to FlightMangaerTest or integration test?